### DISCUSSION TITLE
google-home-off

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [Mariusthvdb/custom-icons#36] [Icon suggestions/requests] [answered]
as google-home, but obviously with the diagonal `-off` line

**Comment by therealmcsparrow** [comment]
Added it in my PR
![google-home-off](https://user-images.githubusercontent.com/46250500/184891015-cbc7f8c4-6ff0-4edf-b25b-18a3d4a81f81.svg)
R

**Comment by therealmcsparrow** [comment]
Sorry changed the direction off the slash to match other icons
![google-home-off](https://user-images.githubusercontent.com/46250500/184892716-a5b4ed41-014f-4f2f-8ca3-ab5c5ffc89c5.svg)
.

**Reply by arallsopp** [reply]
Sorry, late to the party. When I create these for [hass-hue-icons,](https://github.com/arallsopp/hass-hue-icons) I tend to remove a bar of white above the slash. This stops the slash from becoming invisible against filled icons.

Eg: 
<img width="100px" src="https://raw.githubusercontent.com/arallsopp/hass-hue-icons/main/docs/custom_svgs/bulb-group-sultan-lightstrip.svg"> vs <img width="100px" src="https://raw.githubusercontent.com/arallsopp/hass-hue-icons/main/docs/custom_svgs/bulb-group-sultan-lightstrip-off.svg">

**Reply by Mariusthvdb** [reply]
> Sorry, late to the party. When I create these for [hass-hue-icons,](https://github.com/arallsopp/hass-hue-icons) I tend to remove a bar of white above the slash. This stops the slash from becoming invisible against filled icons.
> 

Agreed, it's what MDI does too, so lets keep that please

In this case btw, I like the left and right view of the GH device. I suppose people use them in stereo setups, so we could use a flipped variant for both on and -off

**Reply by arallsopp** [reply]
![google-home-off](https://user-images.githubusercontent.com/2462104/193595308-8eee182b-a22f-45bc-840c-e2d51b63791e.svg)
![google-home-alt-off](https://user-images.githubusercontent.com/2462104/193595310-298cb312-b0d8-4718-ac99-530d1b7417f7.svg)
![google-home-alt](https://user-images.githubusercontent.com/2462104/193595314-2a825f62-7e66-41f9-820b-69de3052ecbe.svg)

Here's three that meet the flipped and off variants then. Out of time for today, so feel free to integrate as you see fit.

**Reply by arallsopp** [reply]
I've added them to a very basic PR at https://github.com/Mariusthvdb/custom-icons/pull/43

**Reply by Mariusthvdb** [reply]
and they are merged, thank you very much

**Reply by Mariusthvdb** [reply]
@arallsopp can you post something, so we can mark that as Answer  ;-)

**Comment by arallsopp** [comment, marked as answer]
| Icon | Purpose |
| :--- | :--- |
|![google-home-off](https://user-images.githubusercontent.com/2462104/193595308-8eee182b-a22f-45bc-840c-e2d51b63791e.svg)|off facing left|
|![google-home-alt-off](https://user-images.githubusercontent.com/2462104/193595310-298cb312-b0d8-4718-ac99-530d1b7417f7.svg)|off facing right|
|![google-home-alt](https://user-images.githubusercontent.com/2462104/193595314-2a825f62-7e66-41f9-820b-69de3052ecbe.svg)|google home facing right|
